asmcomp/asmpackager.ml

```diff
@@ -49,8 +49,9 @@ let read_member_info pack_path file = (
       let (info, crc) = Compilenv.read_unit_info file in
       if info.ui_name <> name
       then raise(Error(Illegal_renaming(name, file, info.ui_name)));
+      (* FIXME should use functions from Compilenv *)
       if info.ui_symbol <>
-         (Compilenv.current_unit_infos()).ui_symbol ^ "__" ^ info.ui_name
+         (Compilenv.current_unit_infos()).ui_symbol ^ "." ^ info.ui_name
```

Comment:
> Unless I missed something, this is Compilenv.make_symbol (Some info.ui_name).

I'm afraid not, because `Compilenv.make_symbol` adds a `caml` prefix.  I'll keep looking in Compilenv...

Reply:
I give up and removed the FIXME.  If it ain't broke...